docs/source/_static/custom.css

```diff
@@ -10,3 +10,21 @@ div.wy-side-nav-search {
   margin: auto;
   display: block;
 }
+
+/* Elevation
+  *
+  * We style box-shadows using Material Design's idea of elevation. These particular numbers are taken from here:
+  *
+  * https://github.com/material-components/material-components-web
+  * https://material-components-web.appspot.com/elevation.html
+  */
+.shadow {
+  border: 1px solid #E0E0E0;
+  box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.2),
+              0px 4px 5px 0px rgba(0, 0, 0, 0.14),
+              0px 1px 10px 0px rgba(0, 0, 0, 0.12);
+}
+
+.rst-content .section>iframe, .rst-content .section>a>iframe {
+  margin-bottom: 24px;
```

Comment:
Also talked with @tgeorgeux and @jzf2101 about upping this to 36px to give a bit more breathing room to everything.